pages/demo.js

```diff
@@ -270,7 +270,7 @@ class FormDemo extends Component {
 const sortedExamples = _.sortBy(examples, ['title']);
 
 const DemoPage = () => (
-  <Page pageKey="demo">
+  <Page pageKey="demo" title="Field Types">
```

Comment:
See comment above: I think you can just pull this from the `pages` object in the `Page` component.